src/components/[guild]/hooks/useAutoStatusUpdate.ts

```diff
@@ -11,39 +11,43 @@ const useAutoStatusUpdate = () => {
   const { id, mutateGuild } = useGuild()
   const { keyPair } = useUserPublic()
 
-  const { data: accesses, mutate: mutateAccess } = useAccess()
-  const { memberships, mutate: mutateMemberships } = useMemberships()
-
-  const roleMemberships = memberships?.find(
-    (membership) => membership.guildId === id
-  )?.roleIds
+  const { onSubmit: onJoin } = useJoin()
```

Comment:
We should just remove this file, no? Membership won't change on it's own so we don't have any data to trigger an automatic update by

Reply:
Oh, yeah, forgot to mention this in the channel, but this feature probably won't work anymore

Reply:
Removed in https://github.com/guildxyz/guild.xyz/pull/1048/commits/f9dd48e934dbf9e0b2fb0e6da995b1851ac0096d!